CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+## [Unreleased]
+### Added
+- README and CHANGELOG files
+### Changed
+- MolField values are always passed to the database as pickled Mol instances
```

Comment:
maybe this one should be rephrased to clarify that get_prep_value now also applies to the ops rhs, or similar?